Android_Development_With_Flutter/Flutter_project/cloud_photos/README.md

```diff
@@ -1,16 +1,37 @@
-# cloud_photos
+<h1 align="center"> Photo Booth </h1>
 
-A new Flutter project.
+## 🎯 About the Project
+* Photo Booth is Flutter Application where you can upload your photos/images on to firebase storage with ease.  
+* It uses Flutter Framework as frontend , backend and Firebase as as server maintaining Databases .
```

Comment:
Line no 5 here can be improved, maybe using tables to display the technologies used here.